indico/htdocs/js/indico/RoomBooking/validation.js

```diff
@@ -75,10 +75,20 @@
         }
 
         // Booked for validator
-        $('#contact_email, #booking_reason').each(function() {
+        var validInput = function($element) {
```

Comment:
We pretty much never use `var foo = function() { ... }'` in new code. What we do use is stuff like this to  make a function global:

global.foo = function foo() { ... };